### DISCUSSION TITLE
Placeholder for "Add New Event"?

### DISCUSSION THREAD

**Discussion by bobbybosler** [liedekef/events-made-easy#28] [Q&A] [answered]
Hey Franky,

I looked through the documentation, but I couldn't find this, so maybe it doesn't exist. Is there any way to return a url (placeholder maybe?) that would take me straight to an "Add New Event" screen (if the user has permission, of course)? I'm looking to add a button that would let my clients quickly get to that screen from the front end.

Currently testing on https://bobbybosler.org/itinerary/

**Comment by liedekef** [comment, marked as answer]
There is no placeholder needed, but the link is a bit hidden :-)
A link like this works (add the http-part of course): admin.php?page=eme-manager&eme_admin_action=add_new_event

**Reply by bobbybosler** [reply]
Awesome. Thanks!
